__tests__/ExpensiMark-HTML-test.js

```diff
@@ -24,7 +24,7 @@ test('Test multi-line bold markdown replacement', () => {
 // Sections starting with > are successfully wrapped with <blockquote></blockquote>
 test('Test quote markdown replacement', () => {
     const quoteTestStartString = '&gt;This is a *quote* that started on a new line.\nHere is a &gt;quote that did not\n```\nhere is a codefenced quote\n>it should not be quoted\n```';
-    const quoteTestReplacedString = '<blockquote>This is a <strong>quote</strong> that started on a new line.</blockquote>Here is a &gt;quote that did not <pre>here&#32;is&#32;a&#32;codefenced&#32;quote<br />&gt;it&#32;should&#32;not&#32;be&#32;quoted</pre>';
+    const quoteTestReplacedString = '<blockquote>This is a <strong>quote</strong> that started on a new line.</blockquote>Here is a &gt;quote that did not <pre>here&#32;is&#32;a&#32;codefenced&#32;quote<br />&gt;it&#32;should&#32;not&#32;be&#32;quoted<br /></pre>';
```

Comment:
Hmm having to add all of these makes me question if we might have purposefully been removing new lines at the end of codefences.

Going to double check this real quick.

Reply:
@bondydaa I think it should not strip the line when doing this conversion, because the context is not known. By that I don't suggest that it should strip it when adding the comment either, given that is the reported issue.

Reply:
@bondydaa One possible purpose that comes to mind is the HTML to MD conversion, which always added a new line. This has been fixed as well.

Reply:
I've elaborated here https://github.com/Expensify/App/issues/14694#issuecomment-1413168389 on the changes

Reply:
and here https://github.com/Expensify/App/issues/14694#issuecomment-1411287892